src/app.html

```diff
@@ -111,88 +108,82 @@ <h1 class="header center -text text-lighten-2 thin">
                                 about the development, deployment and application management processes to make sure they
                                 are doable and sustainable.</p>
                         </div>
                     </div>
-
                     <div class="col s12 m4">
                         <div class="icon-block">
                             <h2 class="center brown-text"><i class="fa fa-graduation-cap fa-lg"></i> </h2>
                             <h5 class="center">I'm an IT polymath</h5>
-
                             <p>A person whose expertise spans a significant number of different subject areas; such a person
                                 is known to draw on complex bodies of knowledge to solve specific problems. As such I get
                                 called upon to deal with integrating multiple technologies that are present in enterprise
                                 systems.
                             </p>
                         </div>
                     </div>
-
                 </div>
                 <div class="row">
-
                     <div class="col s12 m4">
                         <div class="icon-block">
                             <h2 class="center brown-text"><i class="fa fa-bolt fa-lg"></i> </h2>
                             <h5 class="center">I'm agile</h5>
-
                             <p>I worked on projects using the Scrum methodology at varying degrees and work with others to use
                                 the methology effectively.
                             </p>
                         </div>
                     </div>
-
-
                     <div class="col s12 m4">
                         <div class="icon-block">
                             <h2 class="center brown-text"><i class="fa fa-users fa-lg"></i> </h2>
                             <h5 class="center">Cúram expert</h5>
-
                             <p>I've been working with the Curam Enterprise Framework since 2002 and primarily do programming
                                 and integrations with external systems.</p>
                         </div>
                     </div>
-
                     <div class="col s12 m4">
                         <div class="icon-block">
                             <h2 class="center brown-text"><i class="fa fa-cogs fa-lg"></i> </h2>
                             <h5 class="center">DevOps</h5>
-
                             <p>I do that too, I script whereever I can from development to deployment and circle back with ELK
                                 integration and analysis for production.</p>
                         </div>
                     </div>
-
                 </div>
-
+                <div class="row">
+                    <div class="col s12 m4">
+                        <div class="icon-block">
+                            <h2 class="center brown-text"><i class="fa fa-log fa-lg"></i> </h2>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)